docs-src/concepts/what-is-a-state-transition.md

```diff
@@ -0,0 +1,16 @@
+---
+id: what-is-a-state-transition
+title: What is a State Transition?
+sidebar_label: State Transition
+description: A State Transition is a unit of progress by a Workflow Execution.
+tags:
+  - term
+  - explanation
+---
+
+A State Transition is a unit of progress made by a [Workflow Execution](/concepts/what-is-a-workflow-execution).
+Each State Transition is recorded in a persistence store.
+
+Some operations, such as [Activity Heartbeats](/concepts/what-is-an-activity-heartbeat), require only one State Transition.
+Most operations require multiple State Transitions.
+For example, a simple Workflow with two [Activity Tasks](/concepts/what-is-an-activity-task) (and no retries) might need as few as 11 State Transitions.
```

Comment:
```suggestion
For example, a simple Workflow with two [Activity Tasks](/concepts/what-is-an-activity-task) (and no retries) might need as many as 11 State Transitions.
```

"as few" feels like trying to say it's just a small amount. coming from "only one per ST", I expect "as many"

Reply:
I was thinking of retries, but because I specified no retries, the count should indeed be 11. Thanks for the catch!